features/issue.feature

```diff
@@ -31,6 +31,116 @@ Feature: hub issue
       """
     And the exit status should be 0
 
+  Scenario: Fetch issues not assigned to any milestone
+    Given the GitHub API server:
+    """
+    get('/repos/github/hub/issues') {
+      assert :milestone => "none"
+
+      json [
+        { :number => 102,
```

Comment:
For the tests that assert a specific filter parameter has been passed, you don't actually have to return any results. You can return an empty array.